src/plonk/keygen.rs

```diff
@@ -267,12 +270,34 @@ where
     let l0 = vk.domain.lagrange_to_coeff(l0);
     let l0 = vk.domain.coeff_to_extended(l0, Rotation::cur());
 
+    // Compute l_cover(X) which evaluates to 1 for each blinding factor row
+    // and 0 otherwise over the domain.
+    let mut l_cover = vk.domain.empty_lagrange();
```

Comment:
Consider renaming to `l_blind`; `l_cover` doesn't seem a descriptive name to me.